sw/host/opentitantool/Cargo.toml

```diff
@@ -0,0 +1,23 @@
+[package]
+name = "opentitantool"
+version = "0.1.0"
+authors = ["cfrantz <cfrantz@google.com>"]
+edition = "2018"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+anyhow = "1.0"
+thiserror = "1.0"
+opentitanlib = {version="*", path="../opentitanlib"}
```

Comment:
You don't need to specify a version number if you're specifying a path dependency:

```
opentitanlib = { path = "../opentitanlib" }
```